docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -9,7 +9,7 @@ SHELL ["sh", "-lc"]
 # The following `ARG` are mainly used to specify the versions explicitly & directly in this docker file, and not meant
 # to be used as arguments for docker build (so far).
 
-ARG PYTORCH='2.3.0'
+ARG PYTORCH='2.4.0'
 # (not always a valid torch version)
 ARG INTEL_TORCH_EXT='2.3.0'
```

Comment:
can't use `2.4.0` as the image couldn't be built. Don't know if there will be problem at CI runtime however. Let's see